coral/src/app/features/topics/request/TopicRequest.tsx

```diff
@@ -148,24 +140,24 @@ function TopicRequest() {
               )}
               required={true}
             />
-            <Box component={Flexbox} gap={"l1"}>
-              <Box component={FlexboxItem} grow={1} width={"1/2"}>
+            <Box.Flex gap={"l1"}>
+              <Box grow={1} width={"1/2"}>
```

Comment:
Sorry for the nitpick, but the `grow` prop is unnecessary here (and in the other places it is used in this component), I believe.

Reply:
done!